src/screens/AddNewWorkoutScreen/AddNewWorkoutScreen.js

```diff
@@ -1,26 +1,108 @@
 
-import React from 'react';
-import { Button, SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import {Picker} from '@react-native-picker/picker';
+import { Button, KeyboardAvoidingView, SafeAreaView, ScrollView, StyleSheet, Text, TextInput, View } from 'react-native';
 
 const AddNewWorkoutScreen = () => {
 
+  const [workoutTitle, setWorkoutTitle] = useState('')
+  const [excersizeTitle, setExcersizeTitle] = useState('')
```

Comment:
would more consistent to use British English spelling for when making the post request

Reply:
mmmhmm we'll fix the lyzdexia